eq-author-api/src/validation/index.test.js

```diff
@@ -269,6 +271,124 @@ describe("schema validation", () => {
       });
     });
 
+    describe("textarea and textfield answers", () => {
+      [TEXTAREA, TEXTFIELD].forEach(TYPE => {
+        describe("should validate values are between 10 and 2000 inclusive", () => {
+          it(`and return an error for values less than 10 in ${TYPE} answer`, () => {
+            questionnaire = {
+              id: "1",
+              sections: [
+                {
+                  id: "section_1",
+                  title: "section_1",
+                  pages: [
+                    {
+                      id: "page_1",
+                      title: "page title",
+                      answers: [
+                        {
+                          id: "answer_1",
+                          type: TYPE,
+                          label: "Desc",
+                          properties: { maxLength: "9" },
+                        },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            };
```

Comment:
This questionnaire object is set almost the same way on every test. So it could be moved into a before each, and then just mutate the object the way you want it in every test.